src/utils/calculator.ts

```diff
@@ -0,0 +1,39 @@
+import { GatewayOracle } from 'ardrive-core-js';
+import type { ArweaveOracle } from 'ardrive-core-js/lib/arweave_oracle';
+import regression, { DataPoint } from 'regression';
+
+export class Calculator {
+	private static readonly sampleByteVolumes = [
+		100000, // 100KB
```

Comment:
Please use underscores in the number literals to denote commas to enhance readability. Additionally, please use ACTUAL byte counts. i.e.
`
		102_400, // 100KB
		104_857_600, // 100MB
		107_374_182_400 // 10GB
`

Reply:
Hmmm...
1 byte ~ 8 bits
1_000 bytes ~ 1 kilobyte
1_024 bytes ~ 1.024 kilobytes

Am I wrong?

Reply:
You're not wrong when it comes to measuring bytes in decimal format rather than binary. You can find references by searching around "binary vs decimal bytes". On the development side teams tend to use binary. 

Windows OS still uses binary, while latest Ubuntu and MacOS now use decimal.

Our codebase (core-js) was originally measuring bytes in binary when I joined the team, ie: 1 KB = 1_024 bytes.

And it looks like some functions on the surface of arweave-js also use the binary method.

Reply:
So, I should use KiB, MiB, and GiB instead of KB, MB, and GB respectively

Reply:
That sounds good to me. Using the newish KiB, MiB, and GiB formats would be more precise.

Maybe also a comment near the values stating that they're bytes measured in binary. That would be very clear to the reader.

Reply:
Just fixed the sample byte values